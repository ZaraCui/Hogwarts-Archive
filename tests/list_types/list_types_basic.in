ADD COLLECTION tests/data/books_small.csv
LIST TYPES
EXIT

