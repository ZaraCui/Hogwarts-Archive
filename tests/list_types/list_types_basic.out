user: 12 spellbooks successfully added.

user: Charm
Curse
Defense
Herbology
Potion
Transfiguration

user: Ending Archive process.
